app/src/main/java/org/apache/fineract/ui/online/groups/creategroup/AddGroupMemberStepFragment.kt

```diff
@@ -108,17 +108,36 @@ class AddGroupMemberStepFragment : FineractBaseFragment(), Step, NameListAdapter
                         .addErrorCallback { etNewMember.error = it }.check()) {
             if (currentAction == GroupAction.CREATE) {
                 members.add(etNewMember.text.toString())
+                hideAddMemberView()
             } else {
-                members[editItemPosition] = etNewMember.text.toString()
+                if (members[editItemPosition] != etNewMember.text.toString()) {
+                    MaterialDialog.Builder().init(context).apply {
+                        setTitle(getString(R.string.dialog_title_confirm_updation))
+                        setMessage(getString(R.string.dialog_message_confirm_name_updation, members[editItemPosition]))
+                        setPositiveButton(getString(R.string.update)
+                        ) { dialog: DialogInterface?, _ ->
+                            members[editItemPosition] = etNewMember.text.toString()
+                            hideAddMemberView()
+                            dialog?.dismiss()
+                        }
+                        setNegativeButton(getString(R.string.dialog_action_cancel))
+                        createMaterialDialog()
+                    }.run { show() }
```

Comment:
@EGOR-IND it would be better if a function is made for dialog as it is used 2 times and **passing it necessary parameters to either hide or work with Member or Leader**. It would shorten the code and reduces the redundancy.
```suggestion

                 showUpdateConfirmDialog(Int type , other parameters if required)
```

Reply:
Yeah it is getting used two times but in different fragments so for that I have to add this method in fineractbasefragment. Would it be a good idea to do that?

Reply:
Ooh okay i got what you saying, you mean to make a method accessible for both update and delete of group in a fragment. I'll push the changes soon. Thanks for the suggestion.

Reply:
I wasn't there till now but yes i meant that. Looks perfect now.